x-pack/plugins/monitoring/server/config.ts

```diff
@@ -227,4 +73,23 @@ export const configSchema = schema.object({
   }),
 });
 
-export type MonitoringConfig = TypeOf<typeof configSchema>;
+class MonitoringElasticsearchConfig extends ElasticsearchConfig {
```

Comment:
could you remove `MonitoringElasticsearchConfig` from https://github.com/elastic/kibana/blob/2fccd1fe24b299235e22f2b65a7795db0ae2340c/x-pack/plugins/monitoring/server/types.ts#L19 ?

Reply:
Done